aleth-vm/main.cpp

```diff
@@ -239,29 +238,31 @@ int main(int argc, char** argv)
     if (vm.count("input"))
         data = fromHex(vm["input"].as<string>());
     if (vm.count("code"))
-        code = fromHex(vm["code"].as<string>());
+        code = vm["code"].as<string>();
+    if (vm.count("codefile"))
+        codeFile = vm["codefile"].as<string>();
 
     // Read code from input file.
-    if (!inputFile.empty())
+    if (!codeFile.empty())
     {
-        if (!code.empty())
-            cerr << "--code argument overwritten by input file " << inputFile << '\n';
+        if (vm.count("code"))
```

Comment:
this would be more efficient
```suggestion
        if (!code.empty())
```

Reply:
Noted on this.

Reply:
@gumb0 Added on latest commit.